packages/design-system/src/atoms/switch/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { useEffect, useMemo, useRef } from 'react'
+import { Animated, Pressable, StyleSheet, View } from 'react-native'
+import { container } from '../../styles'
+import { useThemeV2 } from '../../hooks'
+
+const CONTAINER_PADDING = 2
+const CONTAINER_WIDTH = 36
+const CONTAINER_HEIGHT = 18
+const THUMB = 14
+const THUMB_CHECKED_POSITION = 18
+
+export interface SwitchProps {
+  checked: boolean
+  onPress: () => void
+  testID?: string
+  disabled?: boolean
+}
+
+const useTrackBackgroundColor = (checked: boolean, disabled?: boolean) => {
```

Comment:
The return type of the function is string, so we don't need `useMemo` here. Strings are compared by symbol (not by pointer), so `useMemo` does nothing here.